features/features.go

```diff
@@ -85,6 +87,7 @@ var features = map[FeatureFlag]bool{
 	MandatoryPOSTAsGET:       false,
 	DisableAuthz2Orders:      false,
 	FasterGetOrderForNames:   false,
+	AccountCreation:          true,
```

Comment:
I think this could be a potentially misleading feature flag name for this branch. How do you feel about using `LegacyRegistrationEnabled` instead?

I want to avoid the term "Account" since ACME v1 uses "Registration". I think it would also be good to emphasize that this is for ACME v1 only and isn't a circuit breaker for ACME v2 account creation.